tests/tests/test_state_scripts.py

```diff
@@ -23,10 +23,25 @@
 from ..common_setup import class_persistent_standard_setup_one_client_bootstrapped
 from .common_update import common_update_procedure
 from ..helpers import Helpers
-from ..MenderAPI import deploy, logger
+from ..MenderAPI import deploy, logger, image
 from .mendertesting import MenderTesting
 from testutils.infra.device import MenderDeviceGroup
 
+
+@pytest.fixture(scope="class")
+def setup_client_state_scripts_update_module(
+    class_persistent_standard_setup_one_client_bootstrapped,
```

Comment:
If you are basing it on `class_persistent`, then it should also be called `class_persistent` I think. Yes, it's long, but it's important when reading that this property is clearly communicated.

Reply:
:+1: